mozci/console/commands/push.py

```diff
@@ -305,10 +307,14 @@ def _list_tasks(tasks):
             previous == PushStatus.BAD
             and current in (PushStatus.GOOD, PushStatus.UNKNOWN)
         ):
+            formatted_date = datetime.datetime.fromtimestamp(
+                push.date, tz=datetime.timezone.utc
+            ).strftime("Pushed at %H hours %M minutes %z")
```

Comment:
```suggestion
            ).strftime("%H:%M")
```

@Archaeopteryx is treeherder always using UTC timezone?
If so, we should always use UTC timezone here too

Reply:
Makes sense if we cannot display it in the timezone used by the client.

Reply:
Please also add the seconds to the timestamp, there can be more than one push per minute.